gradle.properties

```diff
@@ -4,25 +4,24 @@ pluginGroup = intellij
 pluginName = intellij-buf
 pluginRepositoryUrl = https://github.com/bufbuild/intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.5.1
+pluginVersion = 0.6.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 242
+pluginSinceBuild = 251
```

Comment:
Since the breakage occurred in this version, targeting that as the min version for the next release of the plugin.